cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -683,6 +807,17 @@ CALL {
 } IN TRANSACTIONS
 ----
 
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE (a {prop:7})-[r:R]->(b {prop: a.prop})
+----
+a|
+`CREATE` clauses in which a variable introduced in the pattern is also referenced from the same pattern are deprecated.
+
+
```

Comment:
Was this cherry picked to `4.4` ???

Reply:
Yes, in https://github.com/neo4j/neo4j-documentation/pull/1558/files